### PR TITLE
New version: Ubisoft.Connect version 148.1.10975

### LINKED ISSUE #128282: [Package Issue]: Ubisoft.Connect installation fails with "Installer hash does not match"
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

The installation of Ubisoft.Connect fails with `Installer hash does not match.` error.

### Steps to reproduce

Run `winget install --id Ubisoft.Connect` in an elavated shell.

### Actual behavior

The installation fails with:
```powershell
Found Ubisoft Connect [Ubisoft.Connect] Version 147.0.10965
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://static3.cdn.ubi.com/orbit/launcher_installer/UbisoftConnectInstaller.exe
  ██████████████████████████████   224 MB /  224 MB
Installer hash does not match; this cannot be overridden when running as admin
```

### Expected behavior

The installation succeeds.

### Environment

```raw
Windows Package Manager v1.6.3133
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22631.2715
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.21.3133.0
```

### Screenshots and Logs

[WinGet-2023-11-27-22-21-11.727.log](https://github.com/microsoft/winget-pkgs/files/13479883/WinGet-2023-11-27-22-21-11.727.log)

### PR DESCRIPTION
**Pull request by fancybody**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:
Resolves #128282
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/128313)

### CODE REVIEW

File: manifests/u/Ubisoft/Connect/148.0.10969/Ubisoft.Connect.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Ubisoft.Connect
+PackageVersion: 148.0.10969
```
Comment (mdanish-kh) [inline manifests/u/Ubisoft/Connect/148.0.10969/Ubisoft.Connect.installer.yaml:5]:
```suggestion
PackageVersion: 148.1.10975
```

File: manifests/u/Ubisoft/Connect/148.0.10969/Ubisoft.Connect.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Ubisoft.Connect
+PackageVersion: 148.0.10969
```
Comment (mdanish-kh) [inline manifests/u/Ubisoft/Connect/148.0.10969/Ubisoft.Connect.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 148.1.10975
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/u/Ubisoft/Connect` should be renamed from `148.0.10969` -> `148.1.10975`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Version folder also needs to be renamed

https://github.com/microsoft/winget-pkgs/pull/128313#pullrequestreview-1753036170

**Comment by stephengillie** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/128314

**Comment by fancybody** [comment]
@stephengillie 
That commit shows the version as `148.1.0.10975`.
Actual installer is `148.1.10975` (screenshot taken on 2023-Nov-30)
ubisoft_intaller

**Review by stephengillie — APPROVED** [review]

**Pull request merged**